packages/flet/lib/src/services/url_launcher.dart

```diff
@@ -26,9 +27,9 @@ class UrlLauncherService extends FletService {
       case "launch_url":
         return openWebBrowser(args["url"]!,
             webWindowName: args["web_window_name"],
-            webPopupWindow: args["web_popup_window"],
-            windowWidth: args["window_width"],
-            windowHeight: args["window_height"]);
+            webPopupWindow: parseBool(args["web_popup_window"]),
```

Comment:
I don't think we need `parse` here as args are incoming with their right type.

Reply:
You know python doesn't check types. So if a user sends a float instead of an int it will fail. 
Parsing to avoid unnecessary possible errors - doesn't cost us anything.

Reply:
When we integrate pydantic or anything similar, we could then remove these, as we would be 100% sure incoming type will be correct.